worldedit-core/src/main/java/com/sk89q/worldedit/command/NavigationCommands.java

```diff
@@ -57,7 +57,7 @@ public NavigationCommands(WorldEdit worldEdit) {
 
     @Command(
         name = "unstuck",
-        aliases = { "!", "/unstuck" },
```

Comment:
Why have the aliases been removed? They have been around for ages and I don't see a need to remove them now.

Reply:
I can almost guarantee that `!` is used more often than `/unstuck`